apps/web/src/types/Values.ts

```diff
@@ -1,6 +1,6 @@
 export type Values = {
```

Comment:
can we do small case for files?

Reply:
I used pascal case for a component, class, type but camel case for a file contains multiple types.
What are our naming conventions?

Reply:
I think the filename could be all `kebab-case`.